app-portage/portage-utils/portage-utils-9999.ebuild

```diff
@@ -18,50 +16,60 @@ else
 	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
 fi
 
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="openmp +qmanifest +qtegrity static"
+
 RDEPEND="
 	qmanifest? (
-		openmp? (
-			|| (
-				>=sys-devel/gcc-4.2:*[openmp]
-				sys-devel/clang-runtime:*[openmp]
-			)
-		)
-		static? (
-			app-crypt/libb2:=[static-libs]
-			dev-libs/openssl:0=[static-libs]
-			sys-libs/zlib:=[static-libs]
-			app-crypt/gpgme:=[static-libs]
-		)
 		!static? (
 			app-crypt/libb2:=
-			dev-libs/openssl:0=
+			dev-libs/openssl:=
 			sys-libs/zlib:=
 			app-crypt/gpgme:=
 		)
 	)
 	qtegrity? (
-		openmp? (
-			|| (
-				>=sys-devel/gcc-4.2:*[openmp]
-				sys-devel/clang-runtime:*[openmp]
-			)
+		!static? (
+			dev-libs/openssl:=
 		)
+	)"
+DEPEND="${RDEPEND}
+	qmanifest? (
 		static? (
-			dev-libs/openssl:0=[static-libs]
-		)
-		!static? (
-			dev-libs/openssl:0=
+			app-crypt/libb2[static-libs]
+			dev-libs/openssl[static-libs]
+			sys-libs/zlib[static-libs]
+			app-crypt/gpgme[static-libs]
 		)
 	)
-"
-DEPEND="${RDEPEND}"
+	qtegrity? (
+		static? (
+			dev-libs/openssl[static-libs]
+		)
+	)"
+BDEPEND="virtual/pkgconfig"
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	default
+	[[ ${PV} == *9999 ]] && eautoreconf
```

Comment:
@grobian I've added this. In general, if someone uses the live ebuild, we don't want to necessarily rely on the fact that the autotools stuff was regenerated after every merge.

Reply:
I'm fine with that, don't think it holds for all scenarios, but the actions would catch if something doesn't work.  I don't use the 9999, might as well drop it, not sure how useful it is.

Reply:
tell me what you want - I can keep it or remove it

Reply:
keep it, it seems like a nice safeguard